### ISSUE TITLE
Java shim for SET_CURRENT_WINDOW_POSITION fails with geckodriver

### ISSUE THREAD

**Issue by pbi-qfs** [SeleniumHQ/selenium#3893]
## Meta -
OS:  
macOS
Selenium Version:  
3.4.0
Browser:  
Firefox 52esr, 53.0, 54.0b1, 55

## Expected Behavior -
window is positioned correctly

## Actual Behavior -
WebDriverException is throws (FF52esr) or the command is ignored

## Steps to reproduce -

```java
driver.get("http://www.mozilla.com")
driver.manage().window().setPosition(new Point(100,100))
```

Request is incorrectly sent to geckodriver:
```
1493047920997	webdriver::server	DEBUG	→ POST /session/e4c3816b-2f9a-fe43-9d8b1493047920999	Marionette	TRACE	conn0 -> [0,3,"setWindowRect",{"height":null-dc8f00793dc4/window/rect {","width":null,"x":null,"y":null}]
script":"window.screenX = arguments[0]; window.screenY = arguments[1]","args":[100,100]}
1493047920998	geckodriver::marionette	TRACE	→ 68:[0,3,"setWindowRect",{"height":null,"width":null,"x":null,"y":null}]
1493047921000	geckodriver::marionette	TRACE	← [1,3,null,{"x":4,"y":23,"width":1280,"height":877}]
1493047921000	webdriver::server	DEBUG	← 200 OK {"value": {}}
```

Seems to be related to these lines of code: https://github.com/SeleniumHQ/selenium/blob/master/java/client/src/org/openqa/selenium/remote/http/W3CHttpCommandCodec.java#L267-L271

**Comment by andreastt** [comment]
Set Window Rect expects a JSON Object that looks like `{x: <int>, y: <int>, width: <uint>, height: <uint>}`.

**Comment by pbi-qfs** [comment]
Provided fix in PR #3898

**Comment by lmtierney** [comment]
see #3898
